pgjdbc/src/main/java/org/postgresql/PGConnection.java

```diff
@@ -244,6 +244,19 @@ public interface PGConnection {
    */
   PGReplicationConnection getReplicationAPI();
 
+  /**
+   * Change a user's password to the specified new password.
+   * If the specific encryption type is not specified, this method defaults to querying the database server for the server's default password_encryption.
+   * This method does not send the new password in plain text to the server.
+   * Instead, it encrypts the password locally and sends the encoded hash so that the plain text password is never sent on the wire.
+   *
+   * @param user The username of the database user
+   * @param newPassword The new password for the database user
+   * @param encryptionType The type of password encryption to use or null if the database server default should be used.
```

Comment:
Could you add the list of available values for the encryption type?

Reply:
I think we should just list out `md5` and `scram-sha-256` as the other accepted values are really there only to handle server return values (e.g. "on") as aliases for `md5`.